.env

```diff
@@ -0,0 +1 @@
+VITE_AMBITO_URL=https://mercados.ambito.com/home/general3
```

Comment:
Add .env to .gitignore.

*NEVER* push .env to a repository since here's where all your secrets will be added

Reply:
This file should not exist.